internal/test_cases/run_test_case.go

```diff
@@ -36,6 +74,16 @@ func (t *RunTestCase) Run(executable *interpreter_executable.InterpreterExecutab
 	}
 
 	expectedStdout, exitCode, _ := loxapi.Run(t.FileContents)
```

Comment:
```suggestion
	expectedStdout, expectedExitCode, _ := loxapi.Run(t.FileContents)
```

Minor readability improvement - isn't super clear how `exitCode` and `result.ExitCode` are different here.